apps/web/app/features/stacking/user-positions.tsx

```diff
@@ -159,14 +161,22 @@ export function UserPositions({ stacksAddress }: UserPositionsProps) {
           />
         </InfoGrid.Cell>
         <InfoGrid.Cell>
-          <ValueDisplayerWithLoader name="APR" isLoading={fakeLoading} value="~10%" />
+          <ValueDisplayerWithLoader
+            name={<PostLabelHoverCard postKey="historical-yield" label={(content.posts as Record<string, any>)["historical-yield"]?.Title || "APR"} textStyle="label.03" />}
```

Comment:
If these are properly typed there shouldn't be a need to use the `as` keyword to cast them.

Reply:
Resolved in next push